src/RoleObjectFactory.js

```diff
@@ -87,7 +87,7 @@ function createAccessibilityObjectForRole(config) {
   }
 
   if (displayObject.accessible) {
-    // todo: allow changing the role of a DisplayObject
+    parent = displayObject.accessible.parent;
```

Comment:
Since a new AccessibilityObject instance is going to be created and assigned to `displayObject.accessible` later in this function, the existing AccessibilityObject instance will need to be removed from its current parent to avoid having 2 entries in the accessibility tree for the same display object.  `displayObject.accessible.parent && displayObject.accessible.parent.removedChild(displayObject);` should take care of it.